### DISCUSSION TITLE
私用区的“”

### DISCUSSION THREAD

**Discussion by Mjzsl123-github** [lxgw/LxgwWenKai#93] [General]
我挺好奇这是什么东西
![image](https://user-images.githubusercontent.com/110514823/198871914-d884f1ba-cca6-41b1-ad35-08199274a300.jpg)
个人认为，可以把钢筋符号也放进私用区

**Comment by lxgw** [comment]
这个是[「纯纯写作」](http://a.app.qq.com/o/simple.jsp?pkgname=com.drakeet.purewriter&fromcase=70051&g_f=1182517&scenevia=XQYFX)的图标，因为纯纯写作内置了这款字体。
